test-app/package.json

```diff
@@ -11,20 +11,23 @@
     "test": "tests"
   },
   "scripts": {
-    "build": "ember build --environment=production",
+    "build": "npm run tailwind:build && ember build --environment=production",
     "lint": "concurrently 'npm:lint:*(!fix)' --names 'lint:'",
     "lint:fix": "concurrently 'npm:lint:*:fix' --names 'fix:'",
     "lint:hbs": "ember-template-lint . --no-error-on-unmatched-pattern",
-    "lint:js": "eslint . --cache",
     "lint:hbs:fix": "ember-template-lint . --fix --no-error-on-unmatched-pattern",
+    "lint:js": "eslint . --cache",
     "lint:js:fix": "eslint . --fix",
-    "lint:prettier": "prettier -c .",
-    "lint:prettier:fix": "prettier -w .",
     "start": "ember serve",
     "test": "npm-run-all lint test:*",
-    "test:ember": "ember test"
+    "test:ember": "ember test",
+    "tailwind:watch": "npx tailwindcss -c ./config/tailwind/tailwind.config.js -i ./config/tailwind/tailwind-input.css -o ./public/assets/tailwind.css --watch",
+    "tailwind:build": "npx tailwindcss -c ./config/tailwind/tailwind.config.js -i ./config/tailwind/tailwind-input.css -o ./public/assets/tailwind.css",
+    "lint:prettier:fix": "prettier -w .",
+    "lint:prettier": "prettier -c ."
   },
   "dependencies": {
+    "@crowdstrike/tailwind-toucan-base": "^3.3.1",
     "@crowdstrike/ember-toucan-styles": "workspace:../ember-toucan-styles",
```

Comment:
FWIW, I have seen this mostly as `workspace:*`, which would be less brittle towards file layout changes I guess?

Reply:
yeah, I have no preference here 🤷